src/test/scala/com/github/swagger/scala/converter/ModelPropertyParserTest.scala

```diff
@@ -348,6 +348,22 @@ class ModelPropertyParserTest extends AnyFlatSpec with Matchers with OptionValue
     }
   }
 
+  it should "process Array-Model with Scala nonOption Seq" in {
+    val converter = ModelConverters.getInstance()
+    val schemas = converter.readAll(classOf[ModelWStringSeq]).asScala.toMap
+    val model = findModel(schemas, "ModelWStringSeq")
+    model should be(defined)
+    nullSafeList(model.value.getRequired) shouldBe empty
+  }
+
+  it should "process Array-Model with forced required Scala Option Seq" in {
+    val converter = ModelConverters.getInstance()
+    val schemas = converter.readAll(classOf[ModelWOptionStringSeq]).asScala.toMap
+    val model = findModel(schemas, "ModelWOptionStringSeq")
+    model should be(defined)
+    nullSafeList(model.value.getRequired) should not be empty
```

Comment:
can you change this to include a test for the actual value, not just that there is a value?

Reply:
Sure. Fixed now :)